lib/tasks/import_events.rake

```diff
@@ -35,6 +35,8 @@ namespace :import do
           event_row_hash[key] = csv_event_row[event_hash[key]]&.strip || ""
         elsif [:start_date, :end_date].include? key
           event_row_hash[key] = csv_event_row[event_hash[key]]&.strip || nil
```

Comment:
Let's remove the `|| nil` here as it is redundant; `&.strip`already returns nil when something goes wrong.

Reply:
Thanks @Levia I will change that!